src/config/index.js

```diff
@@ -1,4 +1,19 @@
+import { get } from 'lodash';
+
+const getRegistry = () => {
+  const registry = get(window, 'SERVER_FLAGS.registry');
+  if (!registry) {
```

Comment:
we should not forget to setup SERVER_FLAGS for tests and cosmos next time. Or alternatively set a default value here.